other/upgrade.php

```diff
@@ -447,35 +447,19 @@ function load_lang_file()
 		header('Cache-Control: no-cache');
 
 		echo '<!DOCTYPE html>
-<html>
-	<head>
-		<title>SMF Upgrader: Error!</title>
-		<style>
-			body {
-				font-family: sans-serif;
-				max-width: 700px; }
-		
-			h1 {
-				font-size: 14pt; }
-
-			.directory {
-				margin: 0.3em;
-				font-family: monospace;
-				font-weight: bold; }
-		</style>
-	</head>
-	<body>
-		<h1>A critical error has occurred.</h1>
-
-		<p>This upgrader was unable to find the upgrader\'s language file or files.  They should be found under:</p>
-
-		<div class="directory">', dirname($_SERVER['PHP_SELF']) != '/' ? dirname($_SERVER['PHP_SELF']) : '', '/Themes/default/languages</div>
-
-		<p>In some cases, FTP clients do not properly upload files with this many folders. Please double check to make sure you <strong>have uploaded all the files in the distribution</strong>.</p>
-		<p>If that doesn\'t help, please make sure this install.php file is in the same place as the Themes folder.</p>
-		<p>If you continue to get this error message, feel free to <a href="https://support.simplemachines.org/">look to us for support</a>.</p>
-	</body>
-</html>';
+		<html>
```

Comment:
This revers the changes in the recent upgrader PR.

Reply:
It doesn't makes sense to call some text strings if we couldn't find any language files.

Reply:
Yes, but you removed the css changes when you reverted the strings :P